maint/PrepareRelease

```diff
@@ -1,8 +1,7 @@
 #/bin/sh
 
 # Script to prepare the files for building a PCRE2 release. It does some
-# processing of the documentation, detrails files, and creates pcre2.h.generic
-# and config.h.generic (for use by builders who can't run ./configure).
```

Comment:
Despite the comment at the top saying it creates `{config,pcre2}.h.generic`, I can't find any evidence that it has ever done so. I have added a CI job to ensure that these files stay up to date, using the existing `make` recipe in Makefile.am to generate them.